iosApp/iosApp/ContentView.swift

```diff
@@ -51,6 +51,7 @@ struct ContentView: View {
         .onReceive(inspection.notice) { inspection.visit(self, $0) }
         .onAppear {
             Task { await contentVM.loadOnboardingScreens() }
+            Task { await nearbyVM.loadDebugSetting() }
```

Comment:
I'm not seeing the debug info if I just toggle the setting on without restarting the app (and vice versa). I think the nearby VM needs to reload when the setting changes.

Reply:
If you haven't looked at this yet, I based the combined stop and trip details changes off of this branch (partly to use the debug view, partly to use the existing debug loading in nearby VM), and I just fixed it as part of that, so don't worry about getting it working in this PR, it's fine as an interim state on a debug feature flag only we'll see.

Reply:
Okay great, thank you!!